index.html

```diff
@@ -66,5 +72,236 @@ <h3>
         </li>
       </ul>
     </section>
+
+    <!-- Work section -->
+    <div>
+      <works class="works">
```

Comment:
Kindly always use section tags to wrap standalone blocs of code as this will help you organize you code and make it clean and easy to read and good for web accessibility